java-client/src/generated/java/org/opensearch/client/opensearch/indices/Alias.java

```diff
@@ -42,45 +50,46 @@
 import org.opensearch.client.json.ObjectDeserializer;
 import org.opensearch.client.json.PlainJsonSerializable;
 import org.opensearch.client.opensearch._types.query_dsl.Query;
+import org.opensearch.client.util.ApiTypeHelper;
 import org.opensearch.client.util.ObjectBuilder;
 import org.opensearch.client.util.ObjectBuilderBase;
 
-// typedef: indices._types.Alias
+// typedef: indices.Alias
 
 @JsonpDeserializable
+@Generated("org.opensearch.client.codegen.CodeGenerator")
 public class Alias implements PlainJsonSerializable {
+
     @Nullable
     private final Query filter;
 
-    @Nullable
-    private final String indexRouting;
+    @Nonnull
```

Comment:
I believe this is not the shape of the API we support (https://www.elastic.co/guide/en/elasticsearch/reference/7.12/indices-aliases.html)